### ISSUE TITLE
Indico setup list-plugins breaks on new install (Indico 2.2.5, Ubuntu 18)

### ISSUE THREAD

**Issue by alscheuring** [indico/indico#4327]
Hello,
I am trying to do a fresh install on a VM with Ubuntu 18. I did the Apache instructions and the Indico site works fine. 
When I try to add plugins, things go wrong.  The first sigh of trouble is when I run "Indico setup list plugins".  I get a stream of errors (see below. 
Do I need to maybe get rid of Python 2.7? I'm seeing errors about that here and there.

```
(.venv) indico@indicotest:~$ indico setup list-plugins
Traceback (most recent call last):
  File "/opt/indico/.venv/bin/indico", line 8, in <module>
    sys.exit(cli())
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/flask/cli.py", line 586, in main
    return super(FlaskGroup, self).main(*args, **kwargs)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/cli/util.py", line 110, in invoke
    return self._impl.invoke(ctx)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/indico/cli/setup.py", line 187, in list_plugins
    plugin = ep.load()
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2323, in load
    self.require(*args, **kwargs)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2346, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/opt/indico/.venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 783, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (idna 2.9 (/opt/indico/.venv/lib/python2.7/site-packages), Requirement.parse('idna<2.9,>=2.5'), set(['requests']))
```

**Comment by ThiefMaster** [comment]
Looks like some incompatibility within the dependencies of dependencies.
Try `pip install 'idna<2.9'`; that should fix it.

**Comment by alscheuring** [comment]
Sweet. Looks like that command is working now.  Thank you!

**Comment by ThiefMaster** [comment]
reopening since this is something we should fix on our side

**Comment by ThiefMaster** [comment]
just released v2.2.6 which fixes this properly
